src/main/resources/db/test/default.xml

```diff
@@ -0,0 +1,105 @@
+  <TMP_NON_CASH_BENEFIT nonCashBenefitId="1" enrollmentId="1" informationDate="2015-08-04" benefitsFromAnySource="1" snap="1" wic="1" tanfChildCare="1" tanfTransportation="1" otherTanf="1" rentalAssistanceOngoing="1" otherBenefitsSource="1" rentalAssistanceTemp="1" otherBenefitsSourceIdentify="1" dateCreated="2015-08-04" dateUpdated="2015-08-04"/>
+  <TMP_ORGANIZATION organizationId="1" organizationName="FromeCo" dateCreated="2015-08-04" dateUpdated="2015-08-04"/>
+  <TMP_PHYSICAL_DISABILITY physicalDisabilityId="1" enrollmentId="1" informationDate="2015-08-04" response="1" indefiniteAndImpairs="1" documentationOnFile="1" receivingServices="1" dateCreated="2015-08-04" dateUpdated="2015-08-04"/>
+  <TMP_PROJECT projectId="1" projectName="Example Project" continuumProject="1" projectType="1" residentialAffiliation="1" resProjectId="0" trackingMethod="3" targetPopulation="1" dateCreated="2015-08-04" dateUpdated="2015-08-04"/>
+  <TMP_PROJECT_CONTINUUM project_coc_id="1" project_id="1" coc_code="EXMPL" date_created="2015-08-04" date_updated="2015-08-04"/>
+  <TMP_PROJECT_FUNDER funderId="1" projectId="1" funder="1" grantId="1234" startDate="2015-08-04" endDate="2015-08-04" dateCreated="2015-08-04" dateUpdated="2015-08-04"/>
+  <TMP_PROJECT_INVENTORY inventoryId="1" projectCoCId="1" informationDate="2015-08-04" householdType="0" bedType="1" availability="1" unitInventory="10" bedInventory="10" chBedInventory="10" vetBedInventory="10" youthBedInventory="10" youthAgeGroup="1" inventoryStartDate="2015-08-04" inventoryEndDate="2015-08-04" participatingBeds="10" dateCreated="2015-08-04" dateUpdated="2015-08-04"/>
+  <TMP_PROJECT_SITE siteId="1" projectCoCId="1" principalSite="1" geocode="11.1&apos;11.1" address="123 Fake St" city="Fakesville" state="GA" zip="12345" dateCreated="2015-08-04" dateUpdated="2015-08-04"/>
+  <TMP_REFERRAL referralId="1" enrollmentId="1" referralDate="2015-08-04" pathTypeProvided="1" referralOutcome="1" rhyTypeProvided="1" dateCreated="2015-08-04" dateUpdated="2015-08-04"/>
+  <TMP_SERVICE serviceId="1" enrollmentId="1" dateProvided="2015-08-04" pathTypeProvided="1" rhyTypeProvided="1" hopwaTypeProvided="1" ssvfTypeProvided="1" ssvfVaSubTypeProvided="1" ssvfCoordinatingSubTypeProvided="1" ssvfDirectSubTypeProvided="1" ssvfOtherService="1" dateCreated="2015-08-04" dateUpdated="2015-08-04"/>
+  <TMP_SUBSTANCE_ABUSE substanceAbuseId="1" enrollmentId="1" informationDate="2015-08-04" response="1" indefiniteAndImpairs="1" documentationOnFile="1" receivingServices="1" pathHowConfirmed="1" dateCreated="2015-08-04" dateUpdated="2015-08-04"/>
+  <TMP_USER userId="1" externalID="slifty@gmail.com" canRead="1" canWrite="1" canAdmin="1" dateUpdated="2016-05-31"/>
+  <TMP_USER userId="4" externalID="test@gmail.com" canRead="1" canWrite="1" canAdmin="0" dateUpdated="2016-05-31"/>
```

Comment:
Presumably test@gmail.com is someone?  (I don't see anything on the Internet to indicate that it's a special test address.)  We probably shouldn't use it; actually, we shouldn't use any real email addresses in our test dataset, except maybe some special addresses that we control & have dedicated explicitly to this test data.